apiary.apib

```diff
@@ -2927,6 +2927,11 @@ Get a list of project milestones.
                     + Members
                         + open
                         + closed
+                + profit_info (object)
```

Comment:
Just wondering if there's a better name for this since it indicates both profit and loss. Is there a term that combines the two from a project/budget perspective?

Reply:
Alternative: `profitability`

Reply:
In a way, we're visualizing the current "budget progress".

Reply:
I like the `profitability` alternative. 

The problem is: if we change it here, I think we should also change it in the project.info call, before integrators start using this.

Reply:
Do we already return it in `project.info`? Otherwise I'd avoid still changing it without a BC layer. Once we pass it to customers, they can start depending on it, even when it's not documented.

Reply:
I'll just change it to "profitability" in milestones, then.

Reply:
I will also create a ticket to change `profit_info` to `profitability` in projects.info (with a BC layer). Not a priority atm though.

Reply:
https://teamleader.atlassian.net/browse/PRO-1468